packages/docs/base/icons.md

```diff
@@ -0,0 +1,206 @@
+# Iconography
+
+ An icon is a graphic that represents an idea or concept, independent of any particular language, specific words, or phrases. Below are icons that can be used for design projects in Odyssey.
+
+ <figure class="ods-table--figure">
+  <figcaption class="ods-table--figcaption">
+    Icons
+  </figcaption>
+  <table class="ods-table">
+    <thead>
+      <tr>
+        <th scope="column">Icon</th>
+        <th scope="column">Name</th>
+        <th scope="column">Use</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Search
+        </td>
+        <td>
+          To search for something
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Info
+        </td>
+        <td>
+          To get information
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          User
+        </td>
+        <td>
+          To support a user name
+        </td>
+      </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          Delete
+        </td>
+        <td>
+          To delete something
```

Comment:
@joejezowski-okta @arnoldsandoval-okta @johnintrater-okta any thoughts on general phrasing for icon use?

In this example:

>To delete something

VS

>To indicate a destructive action